src/app.ts

```diff
@@ -22,6 +38,8 @@ app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerSpec));
  *               type: string
  */
 
```

Comment:
Why you replace `health-check` with `api-docs`? Let's rollback and place this annotation prior `/healthcheck`.

```suggestion
```

Reply:
Im not sure if understand this correctly. I have 2 paths: "/health-check" and "api-docs". I have made swagger comment before "api-docs" and path "api-docs" located prior to path "health-check". Path "health-check" used just to check if routing is working. 
Both paths ("health-check" and "api-docs") are in code now. I have not removed "health-check"